### ISSUE TITLE
App Background Transparent

### ISSUE THREAD

**Issue by JoaquinDecima** [neutralinojs/neutralinojs#320]
**Expected Behavior**  
When using transparencies in css, the application must be transparent and not have a white background
  
**Actual Behavior**  
Using transparencies or rounded edges you can see a white background

**Steps to Reproduce the Problem**  
 1. Open the CSS
 2. Add the backround rgba (0,0,0,0)
 3. Run the App

**Specifications**  
* NeutralinoJs Version: 1.5.0
* NeutralinoJs CLI version: 1.7.0
* Platform: Linux Mint 20

**Comment by jpguzman** [comment]
This would be a nice option.

**Comment by JoaquinDecima** [comment]
Add Documentation 

https://stackoverflow.com/questions/36994927/gtk3-window-transparent

**Comment by JoaquinDecima** [comment]
@shalithasuranga is this possible?

**Comment by guillaume-leo** [comment]
Is something I really need to build my app, someone found something?

**Comment by shalithasuranga** [comment]
Hey.. all we will add this feature soon.. You can check the roadmap: https://github.com/neutralinojs/proposals

Probably this feature can be used with the v2.0 release. Currently we've prioritized [this](https://github.com/neutralinojs/neutralinojs/issues/395) issue.

**Comment by extorc** [comment]
@shalithasuranga What has been progress on this?
This is a very interesting feature and am really looking forward to this.

**Comment by JoaquinDecima** [comment]
This is something really difficult. Check to be able to do it (Linux which is my domain field) But the problem lies in https://github.com/webview/webview that, or will have to implement GTK4 to be able to give full control of the window (same to apply borders rounded) or should they modify the transparency. From the outside this cannot be done at the moment.

If it is an option (in GTK) make the window transparent, but this applies to the entire window. That is to say that even the texts will be transparent and that is completely useless, if it serves to give a style but it is not really very useful.

**Comment by CS6** [comment]
Looking forward to this feature 🙏

**Comment by InDIOS** [comment]
Hi @shalithasuranga, Can you look what is done [here](https://github.com/MichaelKim/webview/blob/master/webview.hpp)? There is a function called `setBgColor` which could help with the implementation of this feature.
BTW nice job with this project 👌

**Comment by abtonc** [comment]
any updates on this?

**Comment by hschneider** [comment]
I second that, too. It would open up new possibilities for frameless windows.

**Comment by Soahr** [comment]
Hi ! I would also be very interested in this feature

**Comment by lacvietanh** [comment]
Yeah, I really need this feature. This is an image from my other project using Electronjs.. but currently I am working with neutralino because it gives me the lightest and smallest app size.. ❤️
![image](https://github.com/neutralinojs/neutralinojs/assets/72616454/18baf85e-9498-41e5-bb5a-253f31bc40ec)

**Comment by shalithasuranga** [comment]
Closing this since the transparent option is available on latest Neutralinojs versions for GNU/Linux and macOS.
